python/requirements/ml/requirements_tune.txt

```diff
@@ -12,13 +14,12 @@ gluoncv==0.10.1.post0
 gpy==1.10.0
 gym[atari]==0.18.3
 h5py==3.1.0; python_version < '3.7'
-h5py==3.3.0; python_version >= '3.7'
+h5py==3.1.0; python_version >= '3.7'
```

Comment:
so these 2 lines are basically:

h5py==3.1.0?

Reply:
Ah yes good catch, I'll change this